src/.vuepress/config.js

```diff
@@ -2,7 +2,7 @@ const webpack = require('webpack');
 
 const siteDescription =
   'Complete Kuzzle Documentation: Guides, Framework, API, SDKs and officials plugins';
-const siteTitle = 'Kuzzle Documentation';
+const siteTitle = '| Kuzzle Documentation V2';
```

Comment:
This title should be dynamic, you can either check the v1 or the v2 and later any other version that will exists. This should take the value of the current version you are checking

Reply:
I used const siteTitle = `| Kuzzle Documentation ${versionString}`; in the new commit